### PR TITLE
Restore testing and artifact collection in openshift-ansible jobs

### PR DESCRIPTION
**Pull request by vrutkovs**
This would construct API URL on first master, fetch in /tmp/auth_url in
setup container, create cluster-admin user, login and save kubeconfig

This is a follow up for https://github.com/openshift/release/pull/1496#issuecomment-422947388

### CODE REVIEW

File: ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml [hunk 1]
```diff
@@ -156,6 +165,48 @@ objects:
         function teardown() {
           set +e
           touch /tmp/shared/exit
+          echo "Gathering artifacts ..."
```
Comment (smarterclayton) [inline ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml]: This is an exact copy of the other artifact gathering, correct?

File: ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml [hunk 2]
```diff
@@ -125,9 +125,18 @@ objects:
           -i test/ci/inventory \
           test/ci/launch.yml | tee -a /tmp/artifacts/ansible.log
 
-        ansible -vvv \
-          -i test/ci/inventory masters \
-          -m fetch -a 'src=/etc/origin/master/admin.kubeconfig dest=/tmp/shared/ flat=true'
+        # Create a new cluster-admin user
```
Comment (smarterclayton) [inline ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml]: You didn't restore the test container, so this PR isn't "restoring testing".

Comment (vrutkovs) [reply ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml]: Yes

File: ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml (hunk 1 above)
Comment (vrutkovs) [reply ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml]: Yes

**Pull request merged**